homeassistant/components/motionblinds_ble/select.py

```diff
@@ -0,0 +1,79 @@
+"""Select entities for the Motionblinds BLE integration."""
+
+from __future__ import annotations
+
+import logging
+
+from motionblindsble.const import MotionBlindType, MotionSpeedLevel
+from motionblindsble.device import MotionDevice
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import ATTR_SPEED, CONF_MAC_CODE, DOMAIN
+from .entity import MotionblindsBLEEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+PARALLEL_UPDATES = 0
+
+
+SELECT_TYPES: dict[str, SelectEntityDescription] = {
+    ATTR_SPEED: SelectEntityDescription(
+        key=ATTR_SPEED,
+        translation_key=ATTR_SPEED,
+        entity_category=EntityCategory.CONFIG,
+        options=[str(speed_level.value) for speed_level in MotionSpeedLevel],
```

Comment:
We should not use enums from a dependency for the select options, as the translation can be out of sync when the dependency changes the enum. On a version bump, it's easily forgotten to update the translations.

```suggestion
        options=["1", "2", "3"],
```

Reply:
But is the effect not the same if I were to update MotionSpeedLevel and not add translations? Or what happens then if the value gets updated to something that is not in the options list?

Reply:
If the value is not in the options, an error is raised. If the option is not in the translations, no error will be raised, but the front end will not show any translation for that value, which is not a friendly user expercience.

Reply:
Thanks I applied your suggestion.

I understand it is not a friendly user experience, but if I personally had to choose I would rather opt for still seeing the option (that is without translation) and it still working, as opposed to it throwing an error and not working anymore? Or is this done to kind of force a translation to be added since otherwise it will not work?